crates/wasmparser/src/validator/operators.rs

```diff
@@ -3963,7 +3975,10 @@ where
     fn visit_struct_set(&mut self, struct_type_index: u32, field_index: u32) -> Self::Output {
         let field_ty = self.struct_field_at(struct_type_index, field_index)?;
         if !field_ty.mutable {
-            bail!(self.offset, "invalid struct.set: struct field is immutable")
+            bail!(
+                self.offset,
+                "invalid struct `set`: struct field is immutable"
```

Comment:
How come the `struct.set` changed to `struct set`?

Reply:
We're reusing this `bail!` for both `struct.set` and `struct.atomic.set`; failing with a `struct.set` message when the instruction is actually `struct.atomic.set` seemed confusing.